templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -601,7 +601,7 @@ Parameters:
       It cannot start or end with a hyphen (-).
     Type: String
   QSS3KeyPrefix:
-    Default: ''
+    Default: 'quickstart-atlassian-jira'
```

Comment:
QSS3KeyPrefix is expected to end with a `/`
See example here:
https://github.com/aws-quickstart/quickstart-atlassian-jira/blob/develop/templates/quickstart-jira-dc-with-vpc.template.yaml#L630
https://github.com/aws-quickstart/quickstart-atlassian-jira/blob/develop/templates/quickstart-jira-dc-with-vpc.template.yaml#L652